src/components/Synthetics/TradeBox/TradeBoxRows/OneClickTrading.tsx

```diff
@@ -28,7 +28,7 @@ import {
 } from "context/SyntheticsStateContext/selectors/tradeboxSelectors";
 import { useSelector } from "context/SyntheticsStateContext/utils";
 import { SUBACCOUNT_DOCS_URL } from "domain/synthetics/subaccount/constants";
-import CloseIcon from "img/navbutton-close.svg";
+import closeIcon from "img/navbutton-close.svg";
```

Comment:
don't we need `?react` here?

Reply:
Same thing. Previously, we had a universal import syntax whether we needed a URL or a React-wrapped svg. Now we should use:
```
import CloseIcon from "img/navbutton-close.svg?react";
```
if we want a React component, or
```
import closeIcon from "img/navbutton-close.svg";
```
if we need a URL.